cottontaildb_client/cottontaildb_cli.py

```diff
@@ -123,12 +128,59 @@ def schema(client, args):
 
 def entity(client, args):
     command = args.subcommand
+    response = None
     if command == 'drop':
         response = client.drop_entity(args.schema_name, args.entity_name)
-        print(response)
     elif command == 'about':
         response = client.get_entity_details(args.schema_name, args.entity_name)
-        print(response)
+    elif command == 'preview':
+        response = client.sample_entity(args.schema_name, args.entity_name, limit=args.limit)
+    if response is not None:
+        print(format_response(response))
+
+
+def format_response(response):
+    """Formats the response object as a string.
+
+    Args:
+        response (object): The object to format.
```

Comment:
The style for this project is to add type hints in the method signature, however in this case I think it would be better to remove the `object` type hint altogether, since it doesn't add any useful information and causes warnings in some IDEs, or to replace it with `Union[list, object]`.